docker-entrypoint.sh

```diff
@@ -33,6 +33,6 @@ cp /work/uboot-mender/integration-binaries/*.tar* /output
 
 # Beagle Bone Black
 /usr/local/bin/build-uboot-bbb.sh
-cp /work/beaglebone_black*.tar /output
+cp /work/beaglebone_black*.tar* /output
```

Comment:
Why don't we use here explicit .tar.gz?

Reply:
For consistency with the other platforms.  They all use .tar*.  Let me know if you want me to change them all.

Reply:
Ah, ok. I guess they are leftovers from when we had the 00x prefixes. No strong opinion, let's leave it as-is.